SwiftCSV/ParsingState.swift

```diff
@@ -9,8 +9,22 @@
 public enum CSVParseError: Error {
     case generic(message: String)
     case quotation(message: String)
+    case duplicateColumns(columnNames: [String])
+    
+    
+    public static func == (lhs: CSVParseError, rhs: CSVParseError) -> Bool {
+        switch (lhs, rhs) {
+        case (.generic(let message1), .generic(let message2)):
+            return message1 == message2
+        case (.quotation(let message1), .quotation(let message2)):
+            return message1 == message2
+        case (.duplicateColumns(let columns1), .duplicateColumns(let columns2)):
+            return columns1 == columns2
+        default:
+            return false
+        }
+    }
```

Comment:
Sorry, I believe this is dead code a.t.m. and we can remove this?

Reply:
Thank you, has fixed these problems